content/post/elm-at-rakuten/index.md

```diff
@@ -701,30 +882,28 @@ Some consider this to be a benefit as it brings you out of your comfort zone and
 
 But for someone else is a burden and could discourage a team from adopting Elm.
 
-## 5. Content-driven static websites
+[Index](#index) | [Top](#top)
 
-Elm is not a good fit to build static websites that are mostly content-driven. In these cases, an old server-side rendered website can be a better option.
-
-On the other side, if you get to like Elm, it is hard to go back to plain JavaScript/HTML/CSS, so we experimented with Elm for static websites. There are several tools for static site generation. We used *elm-starter*[^elm-starter], a library that transforms an Elm website into a PWA that is also installable, works off-line and works without JavaScript.
-
-[^elm-starter]: These are the most common tools to generate static sites in Elm: [elm-pages](https://package.elm-lang.org/packages/dillonkearns/elm-pages/latest/) | [elmstatic](https://github.com/alexkorban/elmstatic) | [elm-starter](https://github.com/lucamug/elm-starter).
-
-The library *elm-starter* pre-renders pages at build time, making it also a candidate for SEO. The results so far are good, and the Lighthouse scores are promising.
+---
 
-## 6. Some JavaScript and CSS are still necessary
+## 5. Some JavaScript and CSS are still necessary
 
-Ideally, we could build an application by just writing in the Elm language. But if you need to use a third-party library not converted to Elm, we still need to use JavaScript. Doing so means entering again in the realm of possible runtime errors.
+Ideally, we could build an application by just writing in the Elm language. But if you need to use a third-party library not converted to Elm, we still need to use JavaScript. Doing so means entering again into the realm of possible runtime errors.
 
 Elm provides three ways to interact with external libraries: Flags, Ports, and Custom Elements.[^interop] All of them require you to write some JavaScript.
 
 In our case, for example, we must use a JavaScript library for handling payments.
 
-The required CSS, while using **Elm-UI**, is limited. In our applications, we have small snippets of CSS that are mainly tricks to support IE11.
+The required CSS, while using the library _Elm-UI_, is limited. In our applications, we have small snippets of CSS that are mainly tricks to support IE11.
 
 Related to this, Elm is probably not a good fit for short projects that require lots of integration with third-party JavaScript libraries.
 
 [^interop]: Elm provides several methodologies to communicate with JavaScript. Here is [an introduction to JavaScript interoperability](https://guide.elm-lang.org/interop/) with some [examples](https://github.com/elm-community/js-integration-examples).
 
+[Index](#index) | [Top](#top)
+
+---
+
 # Conclusion
 
 We mentioned some of the benefits of coding with Elm, or with a purely functional language for that matter. We also talked about the main issues.
```

Comment:
On the line below (can't comment because not changed in this PR)

```diff
- The benefits are overwhelmingly in the majority, compared to the issues, and this is why we are happy with the choice we made.
+ For us the benefits are overwhelming, compared to the issues, and this is why we are happy with the choice we made.
```